pkg/env/env_test.go

```diff
@@ -64,7 +64,24 @@ func getTestCluster() *v2.Cluster {
 	}
 }
 
-/* func Test_processor_WrapperShell(t *testing.T) {
+func getTestClusterWrapperShell() *v2.Cluster {
```

Comment:
getClusterSpecWithEnv? feel free to describe the function name as clearly as possible.